Extending/Property-Editors/Property-Actions/index.md

```diff
@@ -30,8 +30,51 @@ We use `labelKey` and `labelTokens` to retrieve a localized string that is displ
 
 `isDisabled` is used to disable an Action, which change the visual appearance and prevents interaction. Use this option when an action wouldn't provide any change. In the example above, the action `remove all entries` would not have any impact if there is no entries.
 
-## Getting ready for Property Actions
-A Property Editor needs to be implemented as a Component for it to perform the call to expose its Property Actions.
+## Implement Property Actions
+The implementation of Property Actions vary depending on wether your Property Editor is impemented with a Controller or as a Component.
+
+### Controller Implementation
+
+If your Property Editor is implemented with a Controller, you can use the following approach.
+
+See the following example for implementation of Property Actions in a Controller.
```

Comment:
These two sentences says the same thing, right?
> If your Property Editor is implemented with a Controller, you can use the following approach.

> See the following example for implementation of Property Actions in a Controller.

Can I suggest that we only use one of them?
It could be:
`When your Property Editor is implemented with a Controller, use the following approach for the Property Action:`

Reply:
Yes, I guess you are 100% right!